src/features/HomePage/index.tsx

```diff
@@ -84,4 +255,111 @@ const Image = styled.img`
   right: -4rem;
 `;
 
+const WelcomeNotice = styled.div`
+  align-items: center;
+  background-color: ${palette.purple};
+  border-radius: 10px;
+  box-shadow: 0 0 15px 0 rgba(0, 0, 0, 0.2);
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  height: 16rem;
+  justify-content: center;
+  padding: 4rem;
+`;
+
+const Notices = styled.div`
+  background-color: ${palette.white};
+  border-radius: 10px;
+  box-shadow: 0 0 15px 0 rgba(0, 0, 0, 0.2);
+  box-sizing: border-box;
+  padding: 2rem;
+`;
+
+const Notice = styled(Text)`
+  background-color: ${palette.blueWhite};
+  border-left: 10px white solid;
+  box-shadow: -10px 0 0 0 ${palette.blue};
+  left: 10px;
+  margin-top: 1rem;
+  padding: 1rem;
+  position: relative;
+`;
+
+const RightMiddleContainer = styled.div`
+  flex: 1;
+`;
+
+const ConceptContainer = styled.div`
+  background-color: ${palette.white};
+  border-radius: 10px;
+  box-shadow: 0 0 15px 0 rgba(0, 0, 0, 0.2);
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 2rem;
+`;
+
+const Concepts = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+`;
+
+const Concept = styled.div`
+  background-color: ${palette.blueWhite};
+  display: flex;
+  padding: 1rem;
+`;
+
+const Name = styled(Text)`
+  white-space: nowrap;
+`;
+
+const Equals = styled(Text)`
+  margin: 0 0.5rem;
+`;
+
+const Definition = styled(Text)`
+  margin: 0;
+`;
+
+const BottomContainer = styled.div`
+  background-color: ${palette.blueWhite};
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  padding: 4rem ${OUTER_HORIZONTAL_MARGIN};
+`;
+
+const BottomTitle = styled(Text)`
+  color: ${palette.white};
+`;
+
+const MentorContainer = styled.div`
+  align-items: center;
+  display: flex;
+  gap: 2rem;
+  justify-content: space-between;
+`;
+
+const MentorCards = styled.div`
+  display: flex;
+  gap: 1.5rem;
+`;
+
+const FindMentorContainer = styled.div`
+  align-items: center;
+  background-color: ${palette.purple};
+  border-radius: 10px;
+  box-shadow: 0 0 15px 0 rgba(0, 0, 0, 0.2);
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  max-width: 50%;
+  min-height: 26rem;
+  padding: 4rem;
+`;
+
```

Comment:
This file is too big, it needs to be splitted to multiple components. 

I would take one logical component and create some files for each. something like this 

```react
<container>
    <topcontainer>
      <info/>
    <topcontainer />
    <middlecontainer>
       <welcome  props={props}/>
    <middlecontainer />
    <bottomcontainer>
      <mentorslist props={props} />
    <bottomcontainer />
<container />
```

Reply:
Yeah good point! This shouldn't take too long!

Reply:
Done!